tests/Directory.Build.props

```diff
@@ -40,4 +41,16 @@
   <MicrosoftAspNetCoreMvcTestingVersion>6.0.12</MicrosoftAspNetCoreMvcTestingVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <MicrosoftAspNetCoreMvcTestingVersion>7.0.8</MicrosoftAspNetCoreMvcTestingVersion>
```

Comment:
there is no 8?

Reply:
might be in preview?

Reply:
There is an 8 in preview. Should I include it?

Reply:
If it's just for the tests, yes.